### ISSUE TITLE
Tabs are not accessible with Keyboard

### ISSUE THREAD

**Issue by duranmla** [freenowtech/wave#94]
- `@freenow/wave` version: 1.0.7

### Relevant code

```js
<TabBar>
  <TabBar.Link
    onClick={() => setCurrentTab(tabs.get("online"))}
    selected={currentTab === tabs.get("online")}
  >
    Online (10)
  </TabBar.Link>
  <TabBar.Link
    onClick={() => setCurrentTab(tabs.get("offline"))}
    selected={currentTab === tabs.get("offline")}
  >
    Offline (20)
  </TabBar.Link>
</TabBar>
```

### What you did

Testing the keyboard a11y of my application I realize I can't focus on tabs

### Reproduction

https://www.loom.com/share/b10da983dcfe423aa8a822e436827dc2
https://codesandbox.io/s/wave-playground-tabs-p8m72

### Problem description

Tabs are interactive elements thus they should be reachable using the keyboard

**Comment by duranmla** [comment]
It seems that if we pass `<TabBar.Link as={Link}` it will work but:
- The focus outline gets cropped
- It actually makes sense to say to a "Link" component to render as "Link"?

[Image of the tabs being accessible with Keyboard but with outline cropped](https://user-images.githubusercontent.com/1425162/119655170-905eca80-be29-11eb-8d06-6765c2f43fda.png)

**Comment by nlopin** [comment]
There are [WAI-ARIA recommendations](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel) on how the TabBar should behave with the keyboard.
